MovieApplication/MovieApplication/Models/Movie.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class Movie
+{
+    public int movieID { get; set; }
```

Comment:
ID should be enough here, its not needed to specify again the name of the entity, we do this only when we use reference to another entity, like StudioID